src/GrpcProtos/capture.proto

```diff
@@ -139,6 +139,150 @@ message ApiEvent {
   fixed64 r5 = 9;
 }
 
+message ApiScopeStart {
+  int32 pid = 1;
+  int32 tid = 2;
+  uint64 timestamp_ns = 3;
+
+  fixed64 encoded_name_01 = 4;
```

Comment:
I think we should add a comment explaining the rationale behind the choice of fixed64 to encode string vs using bytes or string.

Reply:
Also, I wonder if it would make sense to extract the "fast-string" functionality into its own message as we are using the same pattern in ApiScopeStartAsync. Do you know if could that be done while maintaining performance? @dpallotti?

Reply:
If I understood it correctly, this will lead to an allocation, which is exactly what we are trying to avoid here.

Reply:
@pierricgimmig Yes, extracting the message defeats the purpose of this (expect when on an Arena).